src/components/super-networked-interactable.js

```diff
@@ -66,24 +68,30 @@ AFRAME.registerComponent("super-networked-interactable", {
 
     this.hand = e.detail.hand;
     this.hand.emit("haptic_pulse", { intensity: "high" });
-    if (this.networkedEl && !NAF.utils.isMine(this.networkedEl)) {
-      if (NAF.utils.takeOwnership(this.networkedEl)) {
-        this.el.setAttribute("body", { type: "dynamic" });
-        this._syncCounterRegistration();
+    if (this.networkedEl) {
+      if (!NAF.utils.isMine(this.networkedEl)) {
+        if (NAF.utils.takeOwnership(this.networkedEl)) {
+          this.el.setAttribute("ammo-body", { type: "dynamic" });
+          this.el.body.forceActivationState(4);
```

Comment:
Same as collision flags - I suggest you export them and then refer to them by name